cloudomate/hoster/vps/blueangelhost.py

```diff
@@ -97,20 +97,16 @@ def get_configuration(self):
     def get_status(self):
         status = super().get_status()
 
-        # Retrieve the vserverid
-        page = self._browser.open(status.clientarea.url)
-        match = re.search(r'vserverid = (\d+)', page.text)
-        identifier = match.group(1)
-
-        millis = int(round(time.time() * 1000))  # Needed for some reason
-        page = self._browser.open('{}?vserverid={}&_={}'.format(self.CLIENT_DATA_URL, identifier, millis))
+        # Get server stats
+        page = self._browser.open('{}&api=json&act=vpsmanage&stats=1'.format(status.clientarea.url))
         data = page.json()
 
-        memory = VpsStatusResource(self._convert_gigabyte(data['memoryused']),
-                                   self._convert_gigabyte(data['memorytotal']))
-        storage = VpsStatusResource(self._convert_gigabyte(data['hddused']), self._convert_gigabyte(data['hddtotal']))
-        bandwidth = VpsStatusResource(self._convert_gigabyte(data['bandwidthused']),
-                                      self._convert_gigabyte(data['bandwidthtotal']))
+        memory = VpsStatusResource(data['info']['ram']['used']/1024.0,
```

Comment:
is it too bad to create a function here to convert mb to gb?
it's more about the readability (no one needs to read the code of convert_to_gb(...)) in order to understand, while /1024 is not as easily understandable

Reply:
done